src/redux/walletconnect.ts

```diff
@@ -198,15 +198,13 @@ const WALLETCONNECT_ADD_URI = 'walletconnect/WALLETCONNECT_ADD_URI';
  */
 const getNativeOptions = async () => {
   const language = 'en'; // TODO use lang from settings
-  let token = null;
-  try {
-    token = await getFCMToken();
-  } catch (error) {
+  const token = await getFCMToken();
```

Comment:
are we safe not catching an error here?

Reply:
Yeah I removed the `throw Error` in `getFCMToken` and `getLocal` returns `value | null`, should be good.